slick.grid.js

```diff
@@ -2346,6 +2346,7 @@ if (typeof Slick === "undefined") {
       lastRenderedScrollTop = scrollTop;
       lastRenderedScrollLeft = scrollLeft;
       h_render = null;
+      trigger(self.onRendered, { startRow: visible.top, endRow: visible.bottom, grid: self })

```

Comment:
How are you planning on using this?

Reply:
this is how this event will be consumed. https://github.com/Microsoft/azuredatastudio/pull/2781

Reply:
i will update this PR and the PR for ads to use this event.

Reply:
This looks fine, but keep in mind that this will be triggered a lot so any listener should be performant.